helper/jfr.go

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package helper
+
+import (
+	"os"
+
+	"github.com/paketo-buildpacks/libpak/sherpa"
+
+	"github.com/paketo-buildpacks/libpak/bard"
+)
+
+type JFR struct {
+	Logger bard.Logger
+}
+
+func (j JFR) Execute() (map[string]string, error) {
+	if val, ok := os.LookupEnv("BPL_JAVA_FLIGHT_RECORDER_ENABLED"); !ok || val != "true" {
+		return nil, nil
+	}
+
+	argList := sherpa.GetEnvWithDefault("BPL_JFR_ARGS", "")
```

Comment:
Whatever we end up doing with the `BPL_*_ENABLED` env variable, we need to make sure this one uses a consistent prefix. That might be updating this one to `BPL_JAVA_FLIGHT_RECORDER_ARGS` or it might be updating the former. They should have the same prefix though.

Reply:
Going with shorter version `BPL_JFR_*` for these